docs/api-table.js

```diff
@@ -1464,6 +1487,31 @@ export const elements = {
     ],
   },
 
+  Select: {
+    required: [],
+    props: [
+      ['auto-focus', 'boolean', 'false', 'Whether the element should receive focus on render.'],
+      [
+        'invalid',
+        'boolean',
+        'false',
+        'Renders the field in an invalid state. Often paired together with `hint` to provide feedback about the error.',
+      ],
+      ['always', 'boolean', '', 'Whether to always show hint.'],
+      ['hint', 'string', '', 'The content to display as the help text.'],
```

Comment:
```suggestion
      ['hint', 'string', '', 'The content to display as the hint.'],
```

I'm just thinking that we might keep the naming consistent (in the previous line we have "whether to always show hint", not "... help text" + the prop is called "hint"). Wdyt?

Reply:
Sure I can change it :)

Reply:
Updated in this commit: [eef7e63](https://github.com/warp-ds/tech-docs/pull/216/commits/eef7e638298d392830bfc81184eb31c19f867b0a)